js/API.js

```diff
@@ -0,0 +1,69 @@
+import {ADDRESS, DEFAULT_USER} from './constants.js';
+
+export const API = {
+  GetUsers: async () => {
+    try{
+      const response = await fetch(`${ADDRESS}/api/users`, ApiOptions.GET());
+      return await response.json();
+    } catch {
+      return DEFAULT_USER.json();
+    }
+  },
+  AddUser: async (userName) => {
+    await fetch(`${ADDRESS}/api/users`, ApiOptions.POST({name: userName}));
+  },
```

Comment:
```suggestion
 async GetUsers() {
    try{
      const response = await fetch(`${ADDRESS}/api/users`, ApiOptions.GET());
      return await response.json();
    } catch {
      return DEFAULT_USER.json();
    }
  },
  async AddUser (userName) {
    await fetch(`${ADDRESS}/api/users`, ApiOptions.POST({name: userName}));
  },
```
화살표 함수보다 object shorthand 메서드를 쓰면 더 간단하게 표현할 수 있는데 화살표 함수를 쓰는 특별한 이유가 있을까 궁금합니다